lib/constants.js

```diff
@@ -1,3 +1,9 @@
 export const SPARK_VERSION = '1.9.0'
 export const MAX_CAR_SIZE = 200 * 1024 * 1024 // 200 MB
 export const APPROX_ROUND_LENGTH_IN_MS = 20 * 60_000 // 20 minutes
+
+export const RPC_REQUEST = new Request('https://api.node.glif.io/', {
+  headers: {
+    authorization: 'Bearer 6bbc171ebfdd78b2644602ce7463c938'
```

Comment:
Can you please confirm that this is a new token, used only by SPARK?

Reply:
Yes, it's a new token associated with my PL email address, used only here.